crypt/src/crypt.cpp

```diff
@@ -106,6 +106,62 @@ static int Crypt_Base64Decode(lua_State* L)
     return 1;
 }
 
+static int Crypt_EncryptXTEA(lua_State* L){
+    DM_LUA_STACK_CHECK(L, 1);
+
+    size_t srclen;
+    const char* src = luaL_checklstring(L, 1, &srclen);
+
+    size_t keylen;
+    const char* key = luaL_checklstring(L, 2, &keylen);
+    if(keylen>16){
```

Comment:
Please use the same code format as the rest of the file:

```cpp
if (keylen > 16)
{
    luaL_error(L, "key should be <=16");
}
```

ie space before parentheses and curly braces on separate lines